src/wp-includes/blocks/index.php

```diff
@@ -49,8 +49,34 @@ function register_core_block_style_handles() {
 		$files          = get_transient( $transient_name );
 		if ( ! $files ) {
 			$files = glob( wp_normalize_path( __DIR__ . '/**/**.css' ) );
+
+			/*
+			 * Make CSS files path relative before storing them in transient.
+			 *
+			 * Remove the `wp-includes` part to avoid issue when moving site (and the database) to another server :
+			 * /path/to/wp-includes/blocks/xxx/yyy.css => blocks/xxx/yyy.css
+			 */
+			$files = array_map(
+				function ( $file ) use ( $includes_path ) {
```

Comment:
```suggestion
				static function ( $file ) use ( $includes_path ) {
```

Per [58323](https://core.trac.wordpress.org/ticket/58323), Use static closures whenever $this is not used to avoid memory leaks